src/config.ts

```diff
@@ -17,6 +17,10 @@ export const TRANSFERS_POLLING_INTERVAL =
   Number(process.env.TRANSFERS_POLLING_INTERVAL) || 5000
 
 export const WEB3_PROVIDER_URL = process.env.WEB3_PROVIDER_URL || 'UNDEFINED'
+export const BLOCK_METADATA_POLLING_INTERVAL =
+  Number(process.env.BLOCK_METADATA_POLLING_INTERVAL) || 5000
+export const BLOCK_METADATA_DEFAULT_MIN_BLOCK_NUMBER =
```

Comment:
probably a dumb q, but would we ever want any other value besides 19546425?

If it's for testing or something, maybe add a comment?  e.g., "you'd only ever set this for testing".

Reply:
probably not, besides testing like you mentioned